foursight_core/react/api/envs.py

```diff
@@ -0,0 +1,82 @@
+import copy
+from typing import Optional
+import os
+from dcicutils import ff_utils
+
+
+class Envs:
+
+    def __init__(self, known_envs: list):
+        # This known_envs should be the list of annotated environment name objects
+        # as returned by app_utils.get_unique_annotated_environment_names, where each
+        # object contains these fields: name, short_name, full_name, public_name, foursight_name
+        self.known_envs = known_envs
+
+    def get_known_envs(self) -> list:
+        return self.known_envs
+
+    def get_known_envs_with_gac_names(self, gac) -> list:
+        known_envs = copy.deepcopy(self.get_known_envs())
+        for known_env in known_envs:
+            known_env["gac_name"] = gac.get_gac_name(known_env["full_name"])
+        return known_envs
+
+    def get_default_env(self) -> str:
+        return os.environ.get("ENV_NAME", "no-default-env")
```

Comment:
Probably a good idea to do
```
DEFAULT_ENV_PLACHOLDER = 'no-default-env'
```
so that testing doesn't have to know an arbitrary string like this and can instead just use `Envs.DEFAULT_ENV_PLACEHOLDER`.

Reply:
All set.